package.json

```diff
@@ -70,7 +71,10 @@
     "luma.gl": "^7.3.2",
     "mapbox-gl": "^1.13.0",
     "mapbox-gl-compare": "^0.4.0",
+    "material-ui-popup-state": "^4.1.0",
+    "moment": "^2.29.4",
     "next": "^12.2.2",
+    "next-compose-plugins": "^2.2.1",
```

Comment:
Same as above. Why is `next-compose-plugins` added again to the packages. Maybe a merge error?